src/netconfig.cpp

```diff
@@ -82,7 +82,7 @@ static void cmdHostname(Dbus& bus, Arguments& args)
     puts(completeMessage);
 }
 
-/** @brief Set default gateway: `getway IP` */
+/** @brief Set default gateway: `gateway IP` */
```

Comment:
I would prefer if You fix this by a separate commit in this pull-request.

Reply:
Done